dspace-api/src/main/java/org/dspace/app/sitemap/GenerateSitemaps.java

```diff
@@ -168,21 +176,19 @@ public static void generateSitemaps(boolean makeHTMLMap,
         AbstractGenerator sitemapsOrg = null;
 
         if (makeHTMLMap) {
-            html = new HTMLSitemapGenerator(outputDir, htmlMapStem + "?map=",
-                                            null);
+            html = new HTMLSitemapGenerator(outputDir, sitemapStem, ".html");
         }
 
         if (makeSitemapOrg) {
-            sitemapsOrg = new SitemapsOrgGenerator(outputDir, sitemapStem
-                + "?map=", null);
+            sitemapsOrg = new SitemapsOrgGenerator(outputDir, sitemapStem, ".xml");
         }
 
         Context c = new Context(Context.Mode.READ_ONLY);
 
         List<Community> comms = communityService.findAll(c);
 
         for (Community comm : comms) {
-            String url = handleURLStem + comm.getHandle();
+            String url = uiURLStem + "/communities/" + comm.getID();
```

Comment:
At this point in time, they don't work. This can be reverted if the handles work, but I'd prefer to have a working solution
I also thought there were plans to remove the requirement to have a handle (so it would become optional)

Reply:
@benbosman : aha, ok. I didn't realize these were not working.  I'm OK with this change, I was mostly asking the reason why it was made.  I do feel handles will be made optional eventually, so you are correct this is a step in that direction.  I'll close out this request as resolved. Thanks!